third-party/CMakeLists.txt

```diff
@@ -8,10 +8,18 @@ target_include_directories(miniz INTERFACE miniz)
 add_library(pffft STATIC pffft/pffft.c)
 target_woof_settings(pffft)
 target_include_directories(pffft INTERFACE pffft)
-target_link_libraries(pffft)
 
 add_library(spng STATIC spng/spng.c)
 target_woof_settings(spng)
 target_compile_definitions(spng PRIVATE SPNG_USE_MINIZ INTERFACE SPNG_STATIC)
 target_include_directories(spng INTERFACE spng)
 target_link_libraries(spng miniz)
+
+add_library(yyjson STATIC yyjson/yyjson.c)
```

Comment:
We have a libyyjson package in Debian. Could we please first try to build against the system package before our own vendored copy?

https://tracker.debian.org/pkg/yyjson

Reply:
MSYS2 doesn't have it. We also need the latest version for better error reporting.

Reply:
What will I have to change to prefer the system libyyjson for the Debian package?

Reply:
> What will I have to change to prefer the system libyyjson for the Debian package?

Quite a lot I think. We can choose other library, e.g. https://github.com/akheron/jansson

Reply:
I'm fine with the library itself. I just don't want to receive bug reports against the Debian package, because I use a code copy and not the system library.

Reply:
Fixed in https://github.com/fabiangreffrath/woof/pull/1982/commits/47038977a74a6c777104ee904fed2e1ea9b1a90d